api/miraie.js

```diff
@@ -81,15 +81,17 @@ const parseHomeDetails = response => {
         id: d.deviceId,
         name: deviceName,
         friendlyName: d.deviceName,
+        modeStatus : '',
         controlTopic: d.topic ? `${d.topic[0]}/control` : null,
         statusTopic: d.topic ? `${d.topic[0]}/status` : null,
         connectionStatusTopic: d.topic ? `${d.topic[0]}/connectionStatus` : null,
-        haStatusTopic: `miraie-ac/${deviceName}/state`,
-        haAvailabilityTopic: `miraie-ac/${deviceName}/availability`,
-        haActionTopic: `miraie-ac/${deviceName}/action`,
-        haCommandTopic: `miraie-ac/${deviceName}/+/set`,
-        haMonthlyPwrTopic: `miraie-ac/${deviceName}/monthly-power-consumption/state`,
-        haDailyPwrTopic: `miraie-ac/${deviceName}/daily-power-consumption/state`
+        haStatusTopic: `/ac/${deviceName}/status`,

```

Comment:
Do you really need to change the topics? It will be breaking change.

Reply:
haModeTopic in miraie.js, mode_stat_t and cmd_t in discovery.js and other changes related to this mode settings in other files.
you can keep `miraie-ac` no need to change to `/ac`
note: action mqtt and related code is removed, since home assistant action supported are cooling, heating, idle and off, and its not of much use